rclcpp/src/rclcpp/qos.cpp

```diff
@@ -16,6 +16,15 @@
 
 #include <rmw/types.h>
 
+namespace
+{
+/// Check if two rmw_time_t have the same values.
+bool equal(const rmw_time_t & left, const rmw_time_t & right)
```

Comment:
```suggestion
bool equal(const rmw_time_t & left, const rmw_time_t & right) const
```

Reply:
This not a method so it can't have a const qualifier at this location.